### ISSUE TITLE
Create a Schema on which Article elements are global to brainspell and which are private. 

### ISSUE THREAD

**Issue by Sharabesh** [OpenNeuroLab/brainspell-neo#60]

**Comment by 62442katieb** [comment]
I think global elements should include information that is useful to all Brainspell users (`atlas`, `space`, `effect`, `locationFields`, and `tagSearch`), that we want stored in the database and accessible to all, while collection-specific elements include information that is only relevant/useful  to a particular context/collection/researcher (key-value pairs, `include`, and reason for inclusion/exclusion).

**Comment by neelsomani** [comment]
Hi @62442katieb, here's the information that I think is associated with a given article. Would you mind looking through this and filling in the blanks/correcting inaccuracies, and adding additional information if I'm missing anything?

* abstract => global
* authors => global
* doi => global
* experiments => broken down below
* metadata => global? (Not sure about this as well. This is essentially just MeSH descriptors for the entire article and votes right now. Might want to rename this field.)
* neurosynthid => vestigial, global
* pmid => global
* reference => global
* title => global
* stereotaxic_space => global (MNI or Talairach)
* number_of_subjects => global

Within experiments, for each experiment:

* flagged_as_inaccurate => global? (Not sure. Do we still want this?)
* title => global?
* caption => global?
* coordinate_locations_and_z_effective => global?
* include => local
* reason_for_inclusion_exclusion => local

I think that I'm certainly missing some information for each experiment. What is atlas, space, effect, etc.? Are these for each experiment?

Also, do users ever need to tag articles, or do they only generally tag experiments within those articles?

**Comment by jbpoline** [comment]
I think we should rename metadata as article-metadata and for-collection-metadata or something like that to make it clearer

**Comment by neelsomani** [comment]
I agree. I think something like article_metadata is probably appropriate. That's a column in the Articles table, which can store MeSH descriptors, number of subjects, etc. (any global data aside from the existing columns)

**Comment by 62442katieb** [comment]
@jbpoline I agree, renaming these meta-data would make things clearer. Perhaps for-database-metadata (things we want to be maintained in the database, for use by all users) and for-collection-metadata (maintained in individual collections, not transferred to the database)?

@neelsomani Thanks for breaking that information down, I definitely missed some things as I was just looking at the variables from the frontend code. Here, I'm considering "global" to refer to "for-database-metadata" and "local" to refer to "for-collection-metadata". I've included my thoughts below, in bold:

>abstract => global
>authors => global
>doi => global
>experiments => broken down below
>metadata => global? **yes, I think we want this maintained on the database level, it's information that people won't likely want/need to be collection-specific and that will be useful/interesting to users regardless of the purpose or intent of a collection.**
>neurosynthid => vestigial, global
>pmid => global
>reference => global
>title => global
>stereotaxic_space => global (MNI or Talairach) **this is "space", defines the coordinate/coordinate space in which results are reported. We've expanded to include "MNI", "Talairach", "other", and "unknown". It is likely to be the same for all experiments of a paper.**
>number_of_subjects => global **In some cases there will be different numbers of experiments for different experiments, in which case this would be an experiment-level variable.**

Within experiments, for each experiment:

>flagged_as_inaccurate => global **I believe that  would be useful information for all users, yes**
>title => global
>caption => global
>coordinate_locations_and_z_effective => global
>include => local
>reason_for_inclusion_exclusion => local

I think that I'm certainly missing some information for each experiment. What is atlas, space, effect, etc.? **`Atlases` are the sources of the tags/descriptors meta-data (Cognitive Paradigm Ontology, Cognitive Atlas, and BrainMap), which we want to be for-database (same reasoning as above for "metadata"). `Space` refers to the stereotaxic space (MNI, Talairach, Other, or Unknown). `Effect` is the type of effect size reported with the coordinates (Z, F, t, r, p).**

Are these for each experiment? **`Atlases` aren't for each experiment, but there are selected descriptors from these atlases that would be chosen for each experiment, yes. `Space` would be the same for all experiments of a given paper, it's unlikely that there would be different stereotaxic spaces used in one paper. @akeshavan right now, in `ViewArticle` there's an option to specify space for each experiment and I think users would only have to do this once per article. It's probably worth maintaining `effect` for each experiment, though.**

Also, do users ever need to tag articles, or do they only generally tag experiments within those articles? **Usually, meta-analyses are done experiment-wise, so I would expect users to only tag experiments. When it comes to `reason_for_inclusion_exclusion`, though, users will likely want to apply an exclusionary criteria to an entire paper.**

**Comment by 62442katieb** [comment]
@jbpoline okay! article_metadata makes sense to me.

**Comment by 62442katieb** [comment]
update: flagged_as_inaccurate is likely no longer necessary
